Assignment-1/React/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>React - bootcamp</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <div id = 'react-root'>
+     
+    </div>
+    <script crossorigin src="https://unpkg.com/react@18/umd/react.development.js"></script>
+    <script crossorigin src="https://unpkg.com/react-dom@18/umd/react-dom.development.js"></script>
+    <script>
+        const element = React.createElement("h1",{className : 'heading'},"Hello from React");
+        const root = ReactDOM.createRoot(document.getElementById('react-root'))
+        root.render(element)
```

Comment:
```suggestion
        const element = React.createElement("h1", {className : "heading"}, "Hello from React");
        const root = ReactDOM.createRoot(document.getElementById("react-root"));
        root.render(element);
```

Minor nitpick on formatting 🙊 to make it more readable:
- Add spaces before arguments
- Try to keep uniform quote styles atleast throughout a single file ("" or '')

Prettier will help you keep a consistent style and formatting.